docs/learn/digital-assets/read-nft-metadata.md

```diff
@@ -0,0 +1,679 @@
+---
+sidebar_label: '🎨 Read LSP8 NFT Metadata'
+sidebar_position: 5
+description: Learn how to read the metadata specific to a LSP8 NFT.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Read LSP8 NFT Metadata
+
+If your digital asset contract is an [LSP8](../../standards/tokens/LSP8-Identifiable-Digital-Asset.md) NFT or Collection, you can fetch the metadata of specific token IDs. There are two different ways to retrieve the metadata of a `tokenId` based on the configuration of the contract.
+
+- **call [`getDataForTokenId(...)`](../../contracts/contracts/LSP8IdentifiableDigitalAsset/extensions/LSP8Enumerable.md#getdatafortokenid)**: By using this method, you retrieve the individual metadata directly on the contract. This is the **default and recommended procedure**, as it can be used regardless of if token IDs follow a numeral sequence, are created individually after contract creation, or if the metadata was updated for a certain subset.
```

Comment:
I think these would be better as 1. and 2. because above you said there are two ways